src/PageHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import { Copy } from './Copy';
+
+// PageHeader is the header at the top of the page, showing the plugin name
```

Comment:
This is great.  This would be the top-level control on the page, provide the header, and then include the other controls? 

This is what I intended this control to become: 
https://github.com/Voter-Science/trc-react/blob/master/src/PluginShell.tsx

but it looks like I never got around to TrcSearch using that. 

Can you rename this PluginShell, and we'll eventually replace that one in the trc-react library?